grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1660,15 +1656,25 @@ def section_hummock(panels):
                     f"(sum(rate({metric('file_cache_miss')}[$__rate_interval])) by (instance)) / (sum(rate({metric('file_cache_latency_count', file_cache_get_filter)}[$__rate_interval])) by (instance))",
                     "file cache miss rate @ {{instance}}",
                 ),
-
+            ],
+        ),
+        panels.timeseries_percentage(
+            "Bloom-Filter Filtering Rate",
+            "",
+            [
                 panels.target(
                     f"1 - (((sum(rate({metric('state_store_read_req_bloom_filter_positive_counts')}[$__rate_interval])) by (job,instance,table_id,type))) / (sum(rate({metric('state_store_read_req_check_bloom_filter_counts')}[$__rate_interval])) by (job,instance,table_id,type)))",
-                    "read req bloom filter filter rate - {{table_id}} - {{type}} @ {{job}} @ {{instance}}",
+                    "{{table_id}} - {{type}} @ {{job}} @ {{instance}}",
                 ),
-
+            ],
+        ),
+        panels.timeseries_percentage(
+            "Bloom-Filter False-Positive Rate",
+            "",
+            [
                 panels.target(
                     f"1 - (((sum(rate({metric('state_store_read_req_positive_but_non_exist_counts')}[$__rate_interval])) by (job,instance,table_id,type))) / (sum(rate({metric('state_store_read_req_bloom_filter_positive_counts')}[$__rate_interval])) by (job,instance,table_id,type)))",
-                    "read req bloom filter false positive rate - {{table_id}} - {{type}} @ {{job}} @ {{instance}}",
+                    "{{table_id}} - {{type}} @ {{job}} @ {{instance}}",
```

Comment:
please remove `1-` because grafana would keep zero when there is no data reported to promethus.

Reply:
No, Grafana won't do that.

with `1-`

<img width="1477" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/10192522/3627b38a-94f6-4464-8ebe-7621e3707456">

without `1-`

<img width="1472" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/10192522/b4609c5f-2657-4155-93e4-d78b2a37c601">

Thus I think it's okay to keep `1-`, because false-positive rate (FP-rate) is more intuitive to me.